src/status_im/contexts/keycard/sign/events.cljs

```diff
@@ -1,34 +0,0 @@
-(ns status-im.contexts.keycard.sign.events
```

Comment:
@alwx why is this ns removed, specifically the `:keycard/sign-hash` event? I'm wondering cause I rebased on top of this branch to work on keycard signing for dapps, but can't find where the keycard signing functionality is anymore.